go.mod

```diff
@@ -88,3 +87,5 @@ require (
 )
 
 replace github.com/mozilla-services/autograph/verifier/contentsignature => ./verifier/contentsignature/
+
+replace github.com/mozilla-services/autograph/crypto11 => ./crypto11/
```

Comment:
I don't think we need this line if we're changing the paths everywhere?

Reply:
Huh, I guess not? (It looks like we also don't need it for `github.com/mozilla-services/autograph/verifier/contentsignature` in that case, which is what I modeled this after.)

Reply:
(I've got a draft up removing that `replace` that I'll mark as ready to review when the tests pass: https://github.com/mozilla-services/autograph/pull/986)

Reply:
Noted - I'll remove this change here.